pmd-lang-test/pom.xml

```diff
@@ -93,6 +93,11 @@
              so that they are automatically available to users
              of the pmd-lang-test module
         -->
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest</artifactId>
+            <scope>test</scope>
```

Comment:
Note, this should be scope compile - see comment above in the pom file.... I'll change that before merging

Reply:
Thanks!